exercises/pig-latin/test/Tests.hs

```diff
@@ -30,6 +30,7 @@ specs = describe "pig-latin" $
         it "word beginning with ch"  $ translate "chair"   `shouldBe` "airchay"
         it "word beginning with qu"  $ translate "queen"   `shouldBe` "eenquay"
         it "word beginning with qu and a preceding consonant" $ translate "square" `shouldBe` "aresquay"
+        it "... but not words beginning with a vowel then qu" $ translate "equal"  `shouldBe` "equalay"
```

Comment:
the name is a bit weird since this is under `describe "some letter clusters are treated like a single consonant"` and it *doesn't* follow the rule. It's a direct contrast to the above case.

Another alternative: Place it under `describe "some letter clusters are treated like a single vowel"`